test/recipes/30-test_evp_data/evpmac_siphash.txt

```diff
@@ -155,7 +155,7 @@ Output = 5150d1772f50834a503e069a973fbd7c
 MAC = SipHash
 Ctrl = size:13
 Key = 000102030405060708090A0B0C0D0E0F
-Result = MAC_BAD_PARAMS
+Result = MAC_INIT_ERROR
 
 # SIPHASH - default values: 2,4 rounds, explicit 13-byte mac (invalid size)
```

Comment:
Add a test that sets the rounds (for the case you just added)

Reply:
Added.